CFC_WebApp/clients/socialmedia/settings.json.sample

```diff
@@ -0,0 +1,4 @@
+{
+    "fb_app_secret":"ask sid",
```

Comment:
I think that it is fine to leave it as settings.json.sample. My main comments are:

1. Maybe we want to have separate settings for each external service (e.g. `facebook_settings.json.sample`). I am not too insistent on this and am willing to accept your current implementation (`settings.json.sample` with service specific prefixes, e.g. `fb_app_secret`).
1. The format of `settings.json.sample` should be the same as the format of `settings.json`. Otherwise, people will replace the keys and then spend a bunch of time trying to figure out why they are getting parse errors. In your case, for example, in `settings.json`, `fb_app_secret` is a child of `client_settings` while in `settings.json.sample`, it is a top level element.